Jarvis/tests/test_default.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+from Jarvis import Jarvis
+from mock import patch
+
+
+class DefaultTest(unittest.TestCase):
+
+    def setUp(self):
+        self.jarvis = Jarvis()
```

Comment:
I could use some help understanding why this is a problem for **Codacy** given it uses the same syntax as as the test_parser.
The tests pass correctly too.
I cannot figure out how to solve the problem and make Codacy happy.